beagle_etl/migrations/0016_assay.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 2.2.10 on 2020-05-15 03:43
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+def add_assay_config(apps, _):
+    Assay = apps.get_model('beagle_etl', 'Assay')
+
+    a = Assay.objects.create(
+        all=[
+            "10X_Genomics_ATAC",
```

Comment:
Do you maybe want to add assays through migration?

Reply:
Yeah, sounds good. I'll add it in.